README.md

```diff
@@ -1,115 +1,38 @@
-# Publish Data
-[![Build Status](https://travis-ci.org/datagovuk/publish_data_beta.svg?branch=master)](https://travis-ci.org/datagovuk/publish_data_beta)
-[![Code Climate](https://codeclimate.com/github/datagovuk/publish_data_beta/badges/gpa.svg)](https://codeclimate.com/github/datagovuk/publish_data_beta)
-[![Stories in Ready](https://badge.waffle.io/datagovuk/publish_data_beta.svg?label=ready&title=Ready)](http://waffle.io/datagovuk/publish_data_beta)
+[![Code Climate](https://codeclimate.com/github/datagovuk/find_data_beta/badges/gpa.svg)](https://codeclimate.com/github/datagovuk/find_data_beta)
+[![Test Coverage](https://codeclimate.com/github/datagovuk/find_data_beta/badges/coverage.svg)](https://codeclimate.com/github/datagovuk/find_data_beta/coverage)
 
-This repository contains the beta-stage data publishing component of data.gov.uk.
+# data.gov.uk Publish
 
-## Usage
+This repository contains the beta-stage publishing component of data.gov.uk
 
-### Ruby version
-This application currently uses ruby v2.4.0. Use [RVM](https://rvm.io/)) or similar to manage your ruby environment and sets of dependencies.
+## Prerequisites
 
-### Installing ruby gems
-To install gems (dependencies) you will need to first install [Bundler](http://bundler.io/)
+You will need to install the following for development.
 
-### Databases
-You will need Postgres and Elasticsearch installed for this to work.
+  * [rbenv](https://github.com/rbenv/rbenv) or similar to manage ruby versions
+  * [bundler](https://rubygems.org/gems/bundler) to manage gems
+  * [elasticsearch](https://www.elastic.co/) search engine
+  * [postgresql](https://www.postgresql.org/) database
 
-On macOS both Postgres and Elasticsearch can be installed using [Homebrew](https://brew.sh/)
+Most of these can be installed with Homebrew on a Mac.
 
-By default elastic is expected to be running on 127.0.0.1:9200 but if it isn't
-you can override the value by exporting ES_HOST=http://.... but make sure the URL
-does not end with a slash.
+## Getting Started
 
-## Running the application
-```
-$ export SECRET_KEY_BASE=...
-$ bundle install
-$ rake db:setup
-$ rails s
-```
+Run the following commands to get started.
 
-## Running tests
-```
-rails spec
-```
+`
+# install dependencies
+bin/setup
 
-## Fake user accounts
-To log in as a fake user, use the credentials in 'seeds.rb' in the 'db' folder.
+# import production data - ctrl-c after ~1000 datasets
+bin/import
```

Comment:
Might it also be worth mentioning `scripts/replication.sh`?

Reply:
It didn't realise this was here - does this replace bin/import + lib/tasks/import.rake? It seems a lot simpler and should achieve the same effect.

Reply:
It should for us, but it won't for anyone outside of our team trying to use it.

Reply:
OK. I think we should try and minimise the number of setup paths we've got and make them robust. I've not found a need for replication.sh - could we lose that one?

I'm also wondering whether we could download a small version of the other CSVs and combine the import.rake with db/seeds.rb so it's all in one obvious place.

But that's also another PR I reckon!

Reply:
The `replication.sh` script allows us to take a copy of the imported data so you can run the application locally with representative data. It quicker and easier than having to manually import and process the data in the CKAN dump files to achieve the same result.

Additionally, it will also replicate datasets that have been manually published or edited, the `bin/import` doesn't do that.

Reply:
We should also consult the git history for context about how certain features and behaviour came to be.

Reply:
It looks like Laurent and Max wrote the setup.sh file, but I bet they didn't know about this script. replication.sh is much faster, requires less code and gives me all the seed data I need.

Also, if the API dump isn't going to continue in the future, this method is going to be more robust. In an effort to simplify, can we remove setup.sh + associated rake tasks?